tasks/scheduler.go

```diff
@@ -219,12 +225,12 @@ func allValidated(tasks []*deliverTxTask) bool {
 
 func (s *scheduler) PrefillEstimates(reqs []*sdk.DeliverTxEntry) {
 	// iterate over TXs, update estimated writesets where applicable
-	for i, req := range reqs {
+	for _, req := range reqs {
 		mappedWritesets := req.EstimatedWritesets
 		// order shouldnt matter for storeKeys because each storeKey partitioned MVS is independent
 		for storeKey, writeset := range mappedWritesets {
 			// we use `-1` to indicate a prefill incarnation
-			s.multiVersionStores[storeKey].SetEstimatedWriteset(i, -1, writeset)
+			s.multiVersionStores[storeKey].SetEstimatedWriteset(req.AbsoluteIndex, -1, writeset)
```

Comment:
would it be easy to add a unit test for this?

Reply:
yeah, good point, we have a simple unit test to assert the proper values in the MultiVersionStore, I'll add one in